atgu/atgu/atgu.py

```diff
@@ -12,7 +12,7 @@
 from hailtop.hail_logging import AccessLogger
 from hailtop.tls import internal_server_ssl_context
 from hailtop.config import get_deploy_config
-from hailtop import aiogoogle
+from hailtop.aiocloud import gcp as aiogoogle
```

Comment:
minor point: I think import with rename is more confusing in the long term than picking a consistent, simple naming system.

I think I'm roughly fine with `hailtop.aiocloud import gcp`. Using `gcp` as a variable name seems rare. Naming the module `aiogoogle` (and naming the other ones `aioazure`, `aioaws`) also seems fine.